### PR TITLE
Fixes #706

### LINKED ISSUE #706: connector/github: API calls do not account for pagination.

### PR DESCRIPTION
**Pull request by givia**

### CODE REVIEW

File: connector/github/github.go [hunk 1]
```diff
@@ -221,40 +221,52 @@ func (c *githubConnector) user(ctx context.Context, client *http.Client) (user,
...
+			if team.Org.Login == org {
+				groups = append(groups, team.Name)
+			}
+		}
+
+		if len(teams) < 100 {
+			break
+		}
+
+		page = page + 1
```
Comment (ericchiang) [inline connector/github/github.go]: Why not use `Link: rel="next"`? https://developer.github.com/v3/orgs/teams/#response-13

Comment (givia) [inline connector/github/github.go]: I think `rel="next"` is a part of response and increasing the page remains unchanged. But we can change the break condition to `rel="last"`.

Comment (givia) [inline connector/github/github.go]: I changed the code to use link header as you suggested.

**Comment by rithujohn191** [comment]
lgtm. tested out the change.

**Pull request merged**